src/main/java/frc/robot/RaspberryPI.java

```diff
@@ -0,0 +1,82 @@
+package frc.robot;
+import edu.wpi.first.networktables.DoubleSubscriber;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class RaspberryPI {
+	private double fps = 0;
+	private NetworkTable table;
+
+	//FPS Calculation
+	private DoubleSubscriber fpsCounter;
+	private DoubleSubscriber cubeYawSubscriber;
+	private DoubleSubscriber cubeDistanceSubscriber;
+	private DoubleSubscriber coneYawSubscriber;
+	private DoubleSubscriber coneDistanceSubscriber;
+	private double lastValue = 0;
```

Comment:
Can we get a more detailed name? Like last value of what? Also it's generally helpful to note down the units in a comment if it's not obvious from the variable name.

Reply:
lastValue is just an incremented constant we receive from Python. We use it to check when the value sent from Python changes, meaning 1 frame has been processed and we record the time for that in order to calculate the FPS.